Snakefile

```diff
@@ -62,7 +62,10 @@ if USE_ENERGY_PLUS:
     use rule prepare_sector_network from pypsa_earth as prepare_sector_network with:
         input:
             network=pathlib.Path(
-                RESDIR_path,
+                "workflow",
+                "pypsa-earth",
+                "results",
+                SECDIR,
```

Comment:
In the next iterations, it would be good to:
1) revise `{}_path` variables to get rid of `_path`
2) reduce chains of strings under `Path()` replacing them by string path variables